src/components/FlightTicket/FlightTicket.tsx

```diff
@@ -0,0 +1,68 @@
+import styles from "./FlightTicket.module.css";
+import { ColorType, LegType } from "./FlightTicket.definitions";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import {
+  faCircleArrowRight,
+  faPlaneArrival,
+  faPlaneDeparture,
+  faPlane,
+} from "@fortawesome/free-solid-svg-icons";
+import type { FlightTicketProps } from "./FlightTicket.definitions";
+
+const FlightTicket = ({
+  date,
+  departingAirport,
+  arrivingAirport,
+  airline,
+  legType,
+  colorVariant,
+}: FlightTicketProps) => {
+  return (
+    <div
+      className={
+        colorVariant == ColorType.RED
+          ? styles.ticketRed
+          : colorVariant == ColorType.BLUE
+          ? styles.ticketBlue
+          : styles.ticketBlue
+      }
+    >
+      <div className={styles.row1}>
+        <div className={styles.date}>{date}</div>
+      </div>
+
+      <div className={styles.row2}>
+        <div className={styles.departingAirport}>{departingAirport}</div>
+        <FontAwesomeIcon
+          icon={faCircleArrowRight}
+          className={styles.arrowIcon}
+        />
+        <div className={styles.arrivingAirport}>{arrivingAirport}</div>
+      </div>
+
+      <div className={styles.row3}>
+        <div className={styles.legTypeWithImage}>
+          <div className={styles.legType}>{legType.toLocaleUpperCase()}</div>
+          {legTypeImage(legType)}
+        </div>
+        <div className={styles.airline}>{airline.toLocaleUpperCase()}</div>
+      </div>
+
+      <div className={styles.row4}>
+        <div>click to see more details</div>
+      </div>
+    </div>
+  );
+};
+
+const legTypeImage = (legType: LegType) => {
```

Comment:
Move this function above the return statement (inside the component function after line 19) and rename it to `renderLegTypeIcon`. If we ever create a function that returns a React element (in this case `<FontAwesomeIcon />`) we want to prefix the function name with `render...`